org.eclipse.jface.text/src/org/eclipse/jface/text/source/SourceViewerConfiguration.java

```diff
@@ -84,9 +84,20 @@ public int getTabWidth(ISourceViewer sourceViewer) {
 	}
 
 	/**
-	 * Returns the undo manager for the given source viewer. This implementation
-	 * always returns a new instance of <code>DefaultUndoManager</code> whose
-	 * history length is set to 25.
+	 * Returns the line space for the given source viewer. This implementation always returns 0.
+	 * 
+	 * @param sourceViewer the source viewer to be configured by this configuration
+	 * @return the line space
+	 * @see org.eclipse.swt.custom.StyledText#setLineSpacing(int)
+	 * @since 3.23
+	 */
+	public int getLineSpace(ISourceViewer sourceViewer) {
```

Comment:
I see "line spacing" in all apps I've tried instead of "line space". Can you please change it to use those de-facto standard name?

Reply:
will do